go/conversation/templates/conversation/bulk_send.html

```diff
@@ -0,0 +1,102 @@
+{% extends "app.html" %}
+{% load crispy_forms_tags %}
+
+{% block content_title %}{{ action.action_display_name }}{% endblock %}
+
+{% block content_actions_right %}
+<div class="pull-right">
+    <button name="schedule" class="btn btn-primary" value="Schedule" data-toggle="modal" data-target="#schedule-modal">{{ action.action_schedule_verb|default:"Schedule" }}</button>
```

Comment:
Yes, we could put it in a base template. I didn't do it here because I remember us discussing to do it as a custom thing here, and to break it out into a reusable module later. But we can also do it in this PR?